src/AddTodoForm.jsx

```diff
@@ -0,0 +1,10 @@
+function AddTodoForm () {
+return(
+    <form>
+    <label htmlFor = "todoTitle"> Title </label>
+    <input id = "todoTitle" type = "text"/>
```

Comment:
add a required attribute

```suggestion
<input id = "todoTitle" type = "text" required/>
```

this means that when you submit the form without any content, it will be validated